x/gov/client/cli/query.go

```diff
@@ -94,6 +94,7 @@ $ %s query gov proposal 1
 func GetCmdQueryProposals() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "proposals",
+		Args:  cobra.NoArgs,
```

Comment:
Do you know why this line is not covered?

Reply:
In the current testing structure for the CLIs, codecov cannot detect that the code is covered. So I cannot increase the coverage right now. But, actually, the tests DOES cover the CLIs.